core/schemata/json-schema/artifacts/multimedia/imageGallery.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "imageGallery",
+  "type": "object",
+  "title": "Image Gallery",
+  "properties": {
+    "imageGallery": {
```

Comment:
`entityImages` ? @setaman We need to find a good naming here.

Reply:
i think we need a call on this. I'm not familiar with all this new props and what they are for

Reply:
@anatoliynovo maybe you can give @setaman an introduction.

Reply:
In short, it is meant to be able to add multiple images to an entity, for example to be able to display them in a slideshow.

Reply:
Ok, DIVA becomes a social network for data